data/extensions/aseprite-theme/dark/theme.xml

```diff
@@ -694,8 +695,9 @@
             <background color="check_hot_face" state="selected" />
             <icon part="pal_options" />
         </style>
-        <style id="new_frame_button" extends="mini_button" />
-        <style id="color_button" extends="mini_button" font="mini" padding-bottom="1" />
```

Comment:
Why is color_button removed from dark theme.xml?

Reply:
Whoops, copy-paste accident. Fixed!